src/entry/aws-console.ts

```diff
@@ -206,6 +207,68 @@ function delay(ms: number) {
   return new Promise( resolve => setTimeout(resolve, ms) );
 }
 
+interface AccountMenuInfo {
+  accountId: string | null;
+  roleName: string | null;
+  ssoRoleName: string | null;
+  userType: 'iam' | 'sso';
+}
+
+const parseAccountMenu = (accountMenu: HTMLElement, prompts: string[]): AccountMenuInfo => {
+  // scrape values from the aws console account menu (top right)
+  // some pages are using an older version of the menu
+  // the old menu prompt includes the ': ', new one does not
+  // old iam: 'Currently active as: '
+  // old sso: 'Account ID: '
+  // new iam: 'Currently active as'
+  // new sso: 'Account ID'
+  const oldAccountPrompt = accountMenu.firstElementChild?.getElementsByTagName('span')[0].textContent || '';
```

Comment:
I didn't know where to test an old account prompt so I'm not 100% sure that this "selector" is still OK.